src/sentry/api/endpoints/project_details.py

```diff
@@ -223,6 +227,10 @@ def put(self, request, project):
             project.name = result['name']
             changed = True
 
+        if result.get('platform'):
```

Comment:
do we want to validate platform? e.g. should it be a choice from sentry.constants.VALID_PLATFORMS